caddy/frankenphp/Caddyfile

```diff
@@ -5,6 +5,10 @@
 		#worker /path/to/your/worker.php
 		{$FRANKENPHP_CONFIG}
 	}
+	
+	servers {
+		enable_full_duplex
+	}
```

Comment:
Could you just remove these lines so we can merge.

IIUC, the bug the workaround was for has been fixed in Go 1.21 and is not necessary anymore.

Reply:
That's not my understanding @dunglas, AFAIK they added EnableDuplex() specifically to allow users to opt in to the "fix".

Reply:
Also, #692 should be reverted.

Reply:
Sorry, I've been pretty sick and just now catching up. I'm a bit confused on where this conversation went and maybe it's because we are all confused?

1. we implemented our own implementation of "duplex" http1.
2. it didn't handle a few edge cases.
3. #686 was a naive attempt to fall back to Go's implementation, which handles the edge cases.
4. it turns out, there were a few other places this was implemented that also don't handle a few edge cases.
5. this PR is a proper attempt to fall back to Go's implementation.

If we want to implement it ourselves, that's fine, but there are quite a number of edge cases we are not handling and it would probably be better to let Go handle it.

Reply:
My suggestion is to follow the same strategy as Go and Caddy:

Remove all our custom workarounds and do nothing by default, and add some docs explaining how to enable full duplex for people encountering the initial issue.

WDYT?

Reply:
Keeping and fixing our workarounds if full duplex isn't enabled (but not if it is enabled) can also be a good idea, but it maybe a bit more difficult to implement.

Reply:
> Remove all our custom workarounds and do nothing by default, and add some docs explaining how to enable full duplex for people encountering the initial issue.

I like it.